### DISCUSSION TITLE
Removing Device Cast Section

### DISCUSSION THREAD

**Discussion by DanielSosa521** [skuethe/MMM-Spotify#152] [Q&A]
Hello! I love this module and would like to fine-tune it to my liking. What I'm missing, is how to remove the text section that tells you, "Listening from : Chrome Browser", or whatever your source is, of course

![](https://github.com/skuethe/MMM-Spotify/blob/master/screenshots/spotify_default.png)

I can remove the text so it's blank, but the icon is still there taking up space. I'd like to remove it so the song title and album can use that space, I would like it that way

I've been looking through the css and config files with no success. Is there any way to do this?

**Comment by skuethe** [comment]
Hey there. Sorry for the late reply, I wasn't tracking new discussions in this repo.

About your question. There is not yet an option to disable this via config.
You can however overwrite the specific CSS within your custom file:

https://docs.magicmirror.builders/modules/customcss.html

```css
#SPOTIFY_DEVICE {
    display: none;
}
```

Not tested, but that should work.
